### PR TITLE
optimize md

### PR DESCRIPTION
**Pull request by SongSun-TG**
Optimize md:
1.add cloud portal for docker environment
2.change to S3 data source
3.fix bugs in queries

### CODE REVIEW

File: demos/guru_scripts/docker/tutorial/4.x/README.md [hunk 1]
```diff
@@ -35,15 +36,6 @@ gsql ddl.gsql
 
 # Load Data 
 Copy [load.gsql](./script/load.gsql) to your container.
```
Comment (SongSun-TG) [inline demos/guru_scripts/docker/tutorial/4.x/README.md:38]: Here change to load data via S3 data source, so no need to upload csv files.

File: demos/guru_scripts/docker/tutorial/4.x/README.md [hunk 2]
```diff
@@ -179,7 +171,7 @@ Copy [q3a.gsql](./script/q3a.gsql) to your container.
 USE GRAPH financialGraph
 
 // create a query
-CREATE OR REPLACE QUERY q3 (datetime low, datetime high, string accntName) SYNTAX v3 {
+CREATE OR REPLACE QUERY q3a (datetime low, datetime high, string accntName) SYNTAX v3 {
```
Comment (mingxiw) [inline demos/guru_scripts/docker/tutorial/4.x/README.md:174]: this query fixed. pull latest.

File: demos/guru_scripts/docker/tutorial/4.x/README.md [hunk 3]
```diff
@@ -114,7 +106,7 @@ Copy [q2a.gsql](./script/q2a.gsql) to your container.
 USE GRAPH financialGraph
 
 # create a query
-CREATE OR REPLACE QUERY q2 (string accntName) SYNTAX v3 {
+CREATE OR REPLACE QUERY q2a (string accntName) SYNTAX v3 {
```
Comment (mingxiw) [inline demos/guru_scripts/docker/tutorial/4.x/README.md:109]: this query fixed. pull latest.

File: demos/guru_scripts/docker/tutorial/4.x/README.md [hunk 4]
```diff
@@ -20,7 +20,8 @@ This GSQL tutorial contains
 
 # Setup Enviroment 
 
-Follow [Docker setup ](https://github.com/tigergraph/ecosys/blob/master/demos/guru_scripts/docker/README.md) to setup your docker enviroment.
+1. Use cloud portal to setup your docker enviroment( TODO: wait release URL).
```
[the remark this answers is not in the document]
Comment (mingxiw) [reply demos/guru_scripts/docker/tutorial/4.x/README.md:23]: correct. This doc serves both on-prem and cloud user. It's a general tutorial. we should just include two way of data source: local csv vs. s3 files.

File: demos/guru_scripts/docker/tutorial/4.x/script/load.gsql [hunk 5]
```diff
@@ -1,16 +1,20 @@
 USE GRAPH financialGraph
 
 DROP JOB load_data
+DROP DATA_SOURCE s3_data_source_0
```
Comment (mingxiw) [inline demos/guru_scripts/docker/tutorial/4.x/script/load.gsql:4]: can you add a section to make s3? btw: is the secret.key public?

Comment (SongSun-TG) [reply demos/guru_scripts/docker/tutorial/4.x/script/load.gsql:4]: Ok, I didn't use valid secret.key and will not leak the key to user.

**Comment by mingxiw** [comment]
provide s3 loading

**Pull request merged**